deploy/helm/templates/node-collector-daemon-set.yaml

```diff
@@ -168,7 +168,7 @@ spec:
             path: /var/log/journal
         - name: logcheckpoints
           hostPath:
-            path: {{ printf "%s/%s" .Values.otel.logs.filestorage.directory .Values.cluster.uid }}
+            path: {{ printf "%s/%s" .Values.otel.logs.filestorage.directory (sha256sum (include "common.cluster-uid" .)) }}
```

Comment:
Using `sha256sum` instead of the UID directly, because it could contain problematic characters.
That was true even before this PR, but now, it's much more likely to happen, since cluster names are more likely to contain white spaces, etc.
Is changing the name of the directory going to cause any issues for log collection after a collector's upgrade?